appveyor.yml

```diff
@@ -32,7 +32,7 @@ install:
   - ps: "$env:faf_version = \"$(&C:\\Miniconda\\python .\\src\\config\\version.py)\".Replace(' ','')"
   - ps: "$env:APPVEYOR_BUILD_VERSION = \"$($env:faf_version)+$($env:APPVEYOR_BUILD_NUMBER)\""
   - ps: "Write-Host \"$($env:faf_version) + $($env:APPVEYOR_BUILD_NUMBER) = $($env:APPVEYOR_BUILD_VERSION)\""
-  - ps: "$env:PYTEST_QT_API=\"pyqt4v2\""
+  - ps: "$env:PYTEST_QT_API=\"PyQt5v2\""
```

Comment:
Yes, thanks for looking at this PR. It certainly is still a lot of work. I started this to see if it can be done.